app/services/param_to_shql.rb

```diff
@@ -0,0 +1,66 @@
+#
+# ParamToShql
+# 
+# Methods concerning the parsing of params[:facet_filter], which has the following format:
```

Comment:
So is this for facet_filter only? Service name sounds like this is more generic stuff.